src/api/__mocks__/mockHelper.js

```diff
@@ -16,3 +16,9 @@ export function setMockResponse(data) {
 export function getMockResponse() {
```

Comment:
i don't think this need to be exported anymore as it's just used within this module